### ISSUE TITLE
Manual Installation

### ISSUE THREAD

**Issue by lfarah** [nubank/NUAnimationKit#12]
```CocoaPods``` and ```Carthage``` are awesome tools and make our life really easier, but there are some devs who still don't know how to use them. 

It would be cool to add the *Manual installation* guide in your ```README.md```. You can take a look at my [iOS Readme Template](https://github.com/awesome-labs/iOS-readme-template) to see how you can do it.

**Comment by felipesabino** [comment]
> there are some devs who still don't know how to use them

Package management is _at least_ a good practice, so if that is the sole reason behind it I would suggest adding links for [`CocoaPods`](https://guides.cocoapods.org/using/using-cocoapods.html) and [`Carthage`](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) howtos/manuals instead. 😉

**Comment by lfarah** [comment]
@felipesabino I agree that package management is a good practice and I love CocoaPods. As I maintain [AwesomeiOS](https://github.com/vsouza/awesome-ios), I look through 30 open source libraries a day and I see that there are people who really don't know how to use ```CocoaPods``` or ```Carthage```, like [this example](https://github.com/MailOnline/ImageViewer/issues/36) or [this example](https://github.com/xhzengAIB/PopMenu/issues/15)
